resilience4j-micrometer/src/main/java/io/github/resilience4j/micrometer/CircuitBreakerMetrics.java

```diff
@@ -71,10 +71,15 @@ public static CircuitBreakerMetrics ofIterable(String prefix, Iterable<CircuitBr
 
     @Override
     public void bindTo(MeterRegistry registry) {
+        final CircuitBreaker.State[] states = CircuitBreaker.State.values();
         for (CircuitBreaker circuitBreaker : circuitBreakers) {
             final String name = circuitBreaker.getName();
-            Gauge.builder(getName(prefix, name, STATE), circuitBreaker, (cb) -> cb.getState().getOrder())
-                    .register(registry);
+            for (CircuitBreaker.State state : states) {
+                Gauge.builder(getName(prefix, name, STATE), circuitBreaker, (cb) -> cb.getState() == state ? 1 : 0)
```

Comment:
Why do we need a gauge metric per state?
Isn't the current state enough? The metric value could still be `cb.getState().getOrder()`
The query which counts the number of CircuitBreakers with label state==open should still be possible or?

Reply:
It would be backward compatible but add more query possibilities.